<launch>
    <arg name="time_steps" default="600"/>
    <arg name="num_inputs" default="24"/>
    <arg name="num_outputs" default="2"/>
    <arg name="controller" default="feed-forward"/>

    <node pkg="ma_evolution" type="argos_launcher.py" name="argos_launcher" output="screen"
          args="/home/matthijs/Documents/ros_ws/src/ma_evolution/argos_worlds/obstacle_light_no_gui_2.argos" />

    <node pkg="ma_evolution" type="argos_experiment_runner.py" name="argos_experiment_runner" output="screen"
          args="$(arg controller) $(arg time_steps) $(arg num_inputs) $(arg num_outputs)" />
</launch>
